src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+// #region card
+
+.card {
+  background-color: #fff;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__image {
+    display: flex;
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    margin-top: 32px;
+  }
+
+  &__text-conteiner {
+    align-self: flex-start;
+    margin: 0 16px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    padding-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+    margin-top: 4px;
+  }
+
+  &__reviwe {
```

Comment:
There is a typo in the class name `&__reviwe`. It should be `&__review` to match the corrected HTML class name.